### DISCUSSION TITLE
How to change the ColumnWidth in lazer?

### DISCUSSION THREAD

**Discussion by RayAdas** [ppy/osu#23559] [Q&A] [answered]
Hi! I just got a windows touch laptop and want to play osu maina mod. But the tracks are too small. There was a config ColumnWidth in skins.ini on the classic OSU can change this. How I can change this in lazer? I didn't see a skin floder.

**Comment by RayAdas** [comment]
I am currently using a stupid method. First, I adjust the skin parameters in classic osu, and then I package it into an osz file before opening it with Lazer.

**Reply by bdach** [reply, marked as answer]
That is the only supported method for now. A better flow for doing this is planned for later.
